river/command/map.zig

```diff
@@ -30,6 +30,7 @@ const modifier_names = [_]struct {
     modifier: u32,
 }{
     .{ .name = "", .modifier = 0 },
+    .{ .name = "none", .modifier = 0 },
```

Comment:
Should this be `None` for consistency with the other names from xkbcommon?

Reply:
Probably.